qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -177,10 +178,26 @@ def __json_decode__(cls, value: Dict[str, Any]) -> "BasisGateLibrary":
 
 
 class FixedFrequencyTransmon(BasisGateLibrary):
-    """A library of gates for fixed-frequency superconducting qubit architectures.
+    r"""A library of gates for fixed-frequency superconducting qubit architectures.
 
     Note that for now this library supports single-qubit gates and will be extended
     in the future.
+
+    Provided gates:
+        - x: :math:`\pi` pulse around the x-axis.
+        - sx: :math:`\pi/2` pulse around the x-axis.
+        - y: :math:`\pi` pulse around the y-axis.
+        - sy: :math:`\pi/2` pulse around the y-axis.
+
+    Parameters:
```

Comment:
Would it make sense to have a section for the default values here as well? Or just cover them in the docstring for `__init__`. There is a little bit of subtlety to them, so I think it would be good to lay them out clearly. In particular, the `amp` parameter sets the guess for x and then sx is set to half that value while the defaults for the other values apply to all pulses.

Reply:
`Parameters` is reserved string for Sphinx sections. Since this uses parameters in different way, I think it's better to have different names, such as `Pulse Parameters`.

Reply:
We can add the defaults to the description of the parameters. See here: [88ca18b](https://github.com/Qiskit/qiskit-experiments/pull/889/commits/88ca18b465007f52f7baedfc74c3791ba517b449) and here [7fc35c4](https://github.com/Qiskit/qiskit-experiments/pull/889/commits/7fc35c4047d795351c09f923b6198eac20225c44)